### PR TITLE
feat: redesign the webdev about page

### LINKED ISSUE #8842: A minor redesign of the web.dev homepage

### PR DESCRIPTION
**Pull request by mamieorine**
Fixes #8842 

Changes proposed in this pull request:

-  Redesign of the web.dev **about** page (https://web.dev/about/)

### CODE REVIEW

**Comment by mamieorine** [comment]
@devnook  I have done redesign of the about page. Please review: https://deploy-preview-9215--web-dev-staging.netlify.app/about/

Figma: https://www.figma.com/file/eXb7xAA7XROYOYNHkHcMJg/%5Bweb.dev%5D-Master?node-id=481%3A18737&t=pE3KlHQApVgmp2o6-0

However,  some roles on the team section are missing for these authors:
  - Una Kravets
  - Ali Spivak
  - Pete LePage
  - Ewa Gasperowicz

Also, for the [raise an issue](https://deploy-preview-9215--web-dev-staging.netlify.app/about/) text link on the _**Spotted an error**_ section, I am not sure which page this button should navigate to.

File: src/site/content/en/about/index.md [hunk 1]
```diff
@@ -1,49 +1,59 @@
...
+    - name: philipwalton
+      role: Performance Lead
+    - name: una
+      role: Lorem ipsum
+    - name: alispivak
+      role: Lorem ipsum
+    - name: petelepage
+      role: Lorem ipsum
+    - name: ewagasperowicz
+      role: Lorem ipsum
```
Comment (devnook) [inline src/site/content/en/about/index.md:41]:
```suggestion
      role: Tech Lead
```

Comment (mamieorine) [reply src/site/content/en/about/index.md]: I have updated your role.

File: src/site/content/en/about/index.md [hunk 2]
```diff
@@ -1,49 +1,59 @@
...
+    - name: rachelandrew
+      role: Content Lead
+    - name: rowan_m
+      role: SPPI Lead
+    - name: andreban
+      role: Infrastructure Lead
+    - name: philipwalton
+      role: Performance Lead
+    - name: una
+      role: Lorem ipsum
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:35]:
```suggestion
      role: CSS and UI lead
```

File: src/site/content/en/about/index.md [hunk 3]
```diff
@@ -1,49 +1,59 @@
...
+    - paragraph: To help you make a case for work on UI or Performance, we have <a href="/tags/case-study/">case studies</a>. Find out how other companies have used metrics such as Core Web Vitals to see real results. And, if that’s quite enough reading for one day we also have a great archive of <a href="/podcasts/">podcasts</a> and other <a href="/shows/">shows</a>, plus recordings of our <a href="/spaces/">Twitter Space</a> sessions with members of the Chrome team.
+    - paragraph: We believe that a web with multiple browser engines is important, and that web sites and applications should work well no matter which browser your visitors use. We know that you care about that too, and so content on this site should have cross-browser status clearly explained. You’ll see that many articles have a component showing browser support—the data comes from our friends at MDN, via the Browser Compat Data project that powers the data on MDN pages.
+theTeam:
+  title: 'The team'
+  subTitle: 'Lorem ipsum dolor sit amet consectetur.'
+  authors:
+    - name: paulkinlan
+      role: DevRel Lead
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:25]:
```suggestion
      role: DevRel lead
```

File: src/site/content/en/about/index.md [hunk 4]
```diff
@@ -1,49 +1,59 @@
...
+    - name: rachelandrew
+      role: Content Lead
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:27]:
```suggestion
      role: Content lead
```

File: src/site/content/en/about/index.md [hunk 5]
```diff
@@ -1,49 +1,59 @@
...
+    - name: rowan_m
+      role: SPPI Lead
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:29]:
```suggestion
      role: SPPI lead
```

File: src/site/content/en/about/index.md [hunk 6]
```diff
@@ -1,49 +1,59 @@
...
+    - name: andreban
+      role: Infrastructure Lead
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:31]:
```suggestion
      role: Infrastructure lead
```

File: src/site/content/en/about/index.md [hunk 7]
```diff
@@ -1,49 +1,59 @@
...
+    - name: philipwalton
+      role: Performance Lead
+    - name: una
+      role: Lorem ipsum
+    - name: alispivak
+      role: Lorem ipsum
+    - name: petelepage
+      role: Lorem ipsum
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:39]:
```suggestion
      role: Advanced apps lead
```

File: src/site/content/en/about/index.md [hunk 8]
```diff
@@ -1,49 +1,59 @@
...
+    - name: ewagasperowicz
+      role: Tech Lead
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:41]:
```suggestion
      role: Tech lead
```

File: src/site/_data/i18n/authors.yml [hunk 9]
```diff
@@ -4419,4 +4419,10 @@ sriramkrishnan:
...
   description:
-    en: 'Tech Lead and Manager on YouTube Web'
\ No newline at end of file
+    en: 'Tech Lead and Manager on YouTube Web'
+
+alispivak:
+  title:
+    en: 'Ali Spivak'
+  description:
+    en: 'Lorem ipsum'
```
Comment (rachelandrew) [inline src/site/_data/i18n/authors.yml:4428]:
```suggestion
    en: 'Member of Chrome DevRel, lead for Extensions.'
```

**Review by rachelandrew — COMMENTED** [review]
I've made a bunch of changes to add titles to the leads and tighten up the copy.

**Review by devnook — APPROVED** [review]

File: src/site/content/en/about/index.md [hunk 10]
```diff
@@ -1,49 +1,59 @@
...
-      link: 'https://www.youtube.com/c/GoogleChromeDevelopers/videos'
-      summary: 'Keep up to date with all that’s new on the web platform with videos from the Chrome team.'
+    We hope web.dev helps you to create high quality web experiences today. The Chrome team is also working to make the web better tomorrow. If you want to learn more about what we are doing, and offer feedback on the features we are developing and contributing to, check out <a href="https://developer.chrome.com/">Chrome Developers</a>.
+  primaryButtonText: 'Chrome Developers'
+  primaryButtonUrl: 'https://developer.chrome.com/'
+coupledColumn:
+  - icon: 'icons/spotted.svg'
+    title: 'Spotted an error?' 
+    summary: |
+      Please <a href="">raise an issue</a> with information about the page and what’s wrong, and we’ll take a look.
```
Comment (rachelandrew) [inline src/site/content/en/about/index.md:54]:
```suggestion
      Please <a href="https://github.com/GoogleChrome/web.dev/issues">raise an issue</a> with information about the page and what’s wrong, and we’ll take a look.
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**